core/controllers/creator_dashboard.py

```diff
@@ -138,7 +137,6 @@ def get(self):
         self.values.update({
             'DEFAULT_TWITTER_SHARE_MESSAGE_DASHBOARD': (
```

Comment:
is this still required? (ditto for line 54)

Reply:
No it is not.
Removed. Thanks